### ISSUE TITLE
[BUG] Remote Connection Via PSN Fails with 1.7.0

### ISSUE THREAD

**Issue by L0WR1D3Rz** [streetpea/chiaki4deck#326]
**Describe the bug**
I followed the process of getting the PSN tokens correctly. I connect on a remote network and chiaki sees my PS5 and says "Remote Connection Via PSN". I click connect and it says for a few seconds establishing connection with console over PSN and then it says Connection over PSN failed closing. For the record, remote play from local network works correctly with chiaki4deck for the same console.

**Debug Log**
[2024-05-13 11:58:45:846846] [I] Logging to file /home/deck/.var/app/io.github.streetpea.Chiaki4deck/data/Chiaki/Chiaki/log/chiaki_session_2024-05-13_11-58-45-846846.log
[2024-05-13 11:58:45:846846] [I] Chiaki Version 1.7.0
[2024-05-13 11:58:45:847847] [I] Trying to use hardware decoder "vaapi"
[2024-05-13 11:58:45:859859] [I] Using hardware decoder "vaapi" with pix_fmt=vaapi
[2024-05-13 11:58:45:860860] [V] Holepunch session state: 0 = [ ]
[2024-05-13 11:58:45:860860] [I] ChiakiOpusEncoder initialized
[2024-05-13 11:58:45:867867] [I] Connected Steam Deck ... gyro online

[2024-05-13 11:58:45:867867] [I] Controller 0 opened: "Microsoft X-Box 360 pad 0 (030079f6de280000ff11000001000000)"
[2024-05-13 11:58:47:039039] [V] chiaki_holepunch_session_create: Created websocket thread
[2024-05-13 11:58:47:039039] [V] chiaki_holepunch_session_create: Waiting for websocket to open...
[2024-05-13 11:58:48:317317] [V] websocket_thread_func: Connected to push notification WebSocket wss://[44-234-164-247-pushcl.np.communication.playstation.net/np/pushNotification](http://44-234-164-247-pushcl.np.communication.playstation.net/np/pushNotification)
[2024-05-13 11:58:48:317317] [V] Holepunch session state: 1 = [ WS_OPEN ]
[2024-05-13 11:58:48:318318] [V] http_create_session: Sending JSON:
{"remotePlaySessions":[{"members":[{"accountId":"me","deviceUniqueId":"me","platform":"me","pushContexts":[{"pushContextId":7XXXXMASKINGTHISTOPROTECT MY DATA"}]}]}]}
[2024-05-13 11:58:48:318318] [V] websocket_thread_func: PING.
[2024-05-13 11:58:48:632632] [V] websocket_thread_func: Received WebSocket frame of length 0 with flags 64
[2024-05-13 11:58:48:632632] [V] websocket_thread_func: Received PONG.
[2024-05-13 11:58:49:062062] [V] websocket_thread_func: Received WebSocket frame of length 568 with flags 1
[2024-05-13 11:58:49:062062] [V] websocket_thread_func: Received WebSocket frame with 568 bytes of payload.
[2024-05-13 11:58:49:062062] [V] {
  "version":"2.1",
  "method":3001,
  "dataType":"psn:sessionManager:sys:remotePlaySession:created",
  "to":{
    "accountId":XXXXMASKINGTHISTOPROTECT MY DATA,
    "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
    "platform":[
      "REMOTE_PLAY"
    ],
    "isSystemOnly":false,
    "toSystem":false
  },
  "body":{
    "data":{
      "customProperties":{
        "from":{
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "onlineId":XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"REMOTE_PLAY"
        }
      },
      "sessionId":"XXXXMASKINGTHISTOPROTECT MY DATA"
    }
  },
  "contexts":[
    {
      "contextId":"XXXXMASKINGTHISTOPROTECT MY DATA",
      "sequenceNumber":1
    }
  ]
}
[2024-05-13 11:58:49:062062] [V] Received notification of type 1
[2024-05-13 11:58:49:067067] [V] websocket_thread_func: Received WebSocket frame of length 763 with flags 1
[2024-05-13 11:58:49:067067] [V] websocket_thread_func: Received WebSocket frame with 763 bytes of payload.
[2024-05-13 11:58:49:067067] [V] {
  "version":"2.1",
  "method":3001,
  "dataType":"psn:sessionManager:sys:rps:members:created",
  "to":{
    "accountId":XXXXMASKINGTHISTOPROTECT MY DATA,
    "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
    "platform":[
      "REMOTE_PLAY"
    ],
    "isSystemOnly":false,
    "toSystem":false
  },
  "body":{
    "data":{
      "customProperties":{
        "from":{
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"REMOTE_PLAY"
        }
      },
      "members":[
        {
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "joinTimestamp":1715594328891,
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"REMOTE_PLAY"
        }
      ],
      "sessionId":"XXXXMASKINGTHISTOPROTECT MY DATA"
    }
  },
  "contexts":[
    {
      "contextId":"XXXXMASKINGTHISTOPROTECT MY DATA9",
      "sequenceNumber":2
    }
  ]
}
[2024-05-13 11:58:49:067067] [V] Received notification of type 2
[2024-05-13 11:58:49:073073] [V] http_create_session: Received JSON:
{"remotePlaySessions":[{"sessionId":"XXXXMASKINGTHISTOPROTECT MY DATA","members":[{"accountId":"XXXXMASKINGTHISTOPROTECT MY DATA","platform":"REMOTE_PLAY","deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA"}]}]}
[2024-05-13 11:58:49:075075] [V] chiaki_holepunch_session_create: Sent holepunch session creation request
[2024-05-13 11:58:49:075075] [V] wait_for_notification: Found notification of type 1
[2024-05-13 11:58:49:075075] [V] chiaki_holepunch_session_create: Holepunch session created.
[2024-05-13 11:58:49:075075] [V] Holepunch session state: 3 = [ WS_OPEN CREATED ]
[2024-05-13 11:58:49:075075] [V] wait_for_notification: Found notification of type 2
[2024-05-13 11:58:49:075075] [V] chiaki_holepunch_session_create: Client joined.
[2024-05-13 11:58:49:075075] [V] Holepunch session state: 11 = [ WS_OPEN CREATED CLIENT_JOINED ]
[2024-05-13 11:58:49:075075] [I] >> Created session
[2024-05-13 11:58:49:075075] [I] Duid: XXXXMASKINGTHISTOPROTECT MY DATA
[2024-05-13 11:58:49:075075] [V] chiaki_holepunch_session_start: Starting holepunch session XXXXMASKINGTHISTOPROTECT MY DATA for device XXXXMASKINGTHISTOPROTECT MY DATA
[2024-05-13 11:58:49:075075] [V] http_start_session: Sending JSON:
{"commandDetail":{"commandType":"remotePlay","duid":"XXXXMASKINGTHISTOPROTECT MY DATA8","messageDestination":"SQS","parameters":{"initialParams":"{\"accountId\":6XXXXMASKINGTHISTOPROTECT MY DATA,\"roomId\":0,\"sessionId\":\"5bfd9bce-c153-4586-a9ea-7d73f7c256fb\",\"clientType\":\"Windows\",\"data1\":\"tgWPf3dAP/ZYS08diW3FHw==\",\"data2\":\"usNIsY4O3zWVv17NI2cTSg==\"}"},"platform":"PS5"}}
[2024-05-13 11:58:50:037037] [V] websocket_thread_func: Received WebSocket frame of length 789 with flags 1
[2024-05-13 11:58:50:037037] [V] http_start_session: Received JSON:
{"commandId":"XXXXMASKINGTHISTOPROTECT MY DATA"}
[2024-05-13 11:58:50:037037] [V] websocket_thread_func: Received WebSocket frame with 789 bytes of payload.
[2024-05-13 11:58:50:037037] [V] Holepunch session state: 27 = [ WS_OPEN CREATED CLIENT_JOINED DATA_SENT ]
[2024-05-13 11:58:50:037037] [V] {
  "version":"2.1",
  "method":3001,
  "dataType":"psn:sessionManager:sys:rps:members:created",
  "to":{
    "accountId":XXXXMASKINGTHISTOPROTECT MY DATA,
    "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
    "platform":[
      "REMOTE_PLAY"
    ],
    "isSystemOnly":false,
    "toSystem":false
  },
  "body":{
    "data":{
      "customProperties":{
        "from":{
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"PROSPERO"
        }
      },
      "members":[
        {
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "joinTimestamp":1715594329754,
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"PROSPERO"
        }
      ],
      "sessionId":"XXXXMASKINGTHISTOPROTECT MY DATA"
    }
  },
  "contexts":[
    {
      "contextId":"XXXXMASKINGTHISTOPROTECT MY DATA,
      "sequenceNumber":3
    }
  ]
}
[2024-05-13 11:58:50:037037] [V] Received notification of type 2
[2024-05-13 11:58:50:038038] [V] wait_for_notification: Found notification of type 2
[2024-05-13 11:58:50:038038] [V] Holepunch session state: 59 = [ WS_OPEN CREATED CLIENT_JOINED DATA_SENT CONSOLE_JOINED ]
[2024-05-13 11:58:50:475475] [V] websocket_thread_func: Received WebSocket frame of length 543 with flags 1
[2024-05-13 11:58:50:475475] [V] websocket_thread_func: Received WebSocket frame with 543 bytes of payload.
[2024-05-13 11:58:50:475475] [V] {
  "version":"2.1",
  "method":3001,
  "dataType":"psn:sessionManager:sys:rps:customData1:updated",
  "to":{
    "accountId"XXXXMASKINGTHISTOPROTECT MY DATA,
    "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA,
    "platform":[
      "REMOTE_PLAY"
    ],
    "isSystemOnly":false,
    "toSystem":false
  },
  "body":{
    "data":{
      "customData1":"XXXXMASKINGTHISTOPROTECT MY DATA",
      "customProperties":{
        "from":{
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":XXXXMASKINGTHISTOPROTECT MY DATA8",
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"PROSPERO"
        }
      },
      "sessionId":"XXXXMASKINGTHISTOPROTECT MY DATA
    }
  }
}
[2024-05-13 11:58:50:475475] [V] Received notification of type 8
[2024-05-13 11:58:50:475475] [V] wait_for_notification: Found notification of type 8
[2024-05-13 11:58:50:475475] [V] Holepunch session state: 123 = [ WS_OPEN CREATED CLIENT_JOINED DATA_SENT CONSOLE_JOINED CUSTOMDATA1_RECEIVED ]
[2024-05-13 11:58:50:475475] [I] >> Started session
[2024-05-13 11:58:55:478478] [V] websocket_thread_func: PING.
[2024-05-13 11:58:55:792792] [V] websocket_thread_func: Received WebSocket frame of length 0 with flags 64
[2024-05-13 11:58:55:792792] [V] websocket_thread_func: Received PONG.
[2024-05-13 11:58:56:519519] [V] websocket_thread_func: Received WebSocket frame of length 1239 with flags 1
[2024-05-13 11:58:56:519519] [V] websocket_thread_func: Received WebSocket frame with 1239 bytes of payload.
[2024-05-13 11:58:56:520520] [V] {
  "version":"2.1",
  "method":3001,
  "dataType":"psn:sessionManager:sys:rps:sessionMessage:created",
  "to":{
    "accountId":XXXXMASKINGTHISTOPROTECT MY DATA
    "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
    "platform":[
      "REMOTE_PLAY"
    ],
    "isSystemOnly":false,
    "toSystem":false
  },
  "body":{
    "data":{
      "customProperties":{
        "from":{
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"PROSPERO"
        }
      },
      "sessionId":"XXXXMASKINGTHISTOPROTECT MY DATA",
      "sessionMessage":{
        "channel":"remote_play:1",
        "payload":"ver=1.0, type=text, body={\"action\":\"OFFER\",\"reqId\":9,\"error\":0,\"connRequest\":{\"sid\":7022,\"peerSid\":0,\"skey\":\"CkHgbyc+Bo8KWAVOzShnHQ==\",\"natType\":2,\"candidate\":[{\"type\":\"STUN\",\"addr\":\"213.55.240.170\",\"mappedAddr\":\"0.0.0.0\",\"port\":56830,\"mappedPort\":0},{\"type\":\"STATIC\",\"addr\":\"213.55.240.170\",\"mappedAddr\":\"0.0.0.0\",\"port\":9303,\"mappedPort\":0},{\"type\":\"LOCAL\",\"addr\":\"192.168.1.7\",\"mappedAddr\":\"0.0.0.0\",\"port\":9303,\"mappedPort\":0}],\"defaultRouteMacAddr\":\"XXXXMASKINGTHISTOPROTECT MY DATA\",\"localPeerAddr\":{\"accountId\":\"XXXXMASKINGTHISTOPROTECT MY DATA\",\"platform\":\"PROSPERO\"},\"localHashedId\":\"XXXXMASKINGTHISTOPROTECT MY DATA=\"}}"
      }
    }
  }
}
[2024-05-13 11:58:56:520520] [V] Received notification of type 16
[2024-05-13 11:58:56:520520] [V] wait_for_notification: Found notification of type 16
[2024-05-13 11:58:56:520520] [E] Type field wasn't LOCAL or STATIC.
[2024-05-13 11:58:56:520520] [E] session_message_parse: Unexpected JSON schema for holepunch session message.
[2024-05-13 11:58:56:520520] [V] {
  "action":"OFFER",
  "reqId":9,
  "error":0,
  "connRequest":{
    "sid":7022,
    "peerSid":0,
    "skey":"XXXXMASKINGTHISTOPROTECT MY DATA==",
    "natType":2,
    "candidate":[
      {
        "type":"STUN",
        "addr":"XXXXMASKINGTHISTOPROTECT MY DATA",
        "mappedAddr":"0.0.0.0",
        "port":56830,
        "mappedPort":0
      },
      {
        "type":"STATIC",
        "addr":"XXXXMASKINGTHISTOPROTECT MY DATA",
        "mappedAddr":"0.0.0.0",
        "port":9303,
        "mappedPort":0
      },
      {
        "type":"LOCAL",
        "addr":"XXXXMASKINGTHISTOPROTECT MY DATA",
        "mappedAddr":"0.0.0.0",
        "port":9303,
        "mappedPort":0
      }
    ],
    "defaultRouteMacAddr":"XXXXMASKINGTHISTOPROTECT MY DATA",
    "localPeerAddr":{
      "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
      "platform":"PROSPERO"
    },
    "localHashedId":"XXXXMASKINGTHISTOPROTECT MY DATA="
  }
}
[2024-05-13 11:58:56:520520] [E] Failed to parse holepunch session message
[2024-05-13 11:58:56:520520] [E] chiaki_holepunch_session_punch_holes: Failed to wait for OFFER holepunch session message.
[2024-05-13 11:58:56:520520] [E] !! Failed to punch hole for control connection.
[2024-05-13 11:58:57:112112] [V] websocket_thread_func: Received WebSocket frame of length 733 with flags 1
[2024-05-13 11:58:57:112112] [V] websocket_thread_func: Received WebSocket frame with 733 bytes of payload.
[2024-05-13 11:58:57:112112] [V] {
  "version":"2.1",
  "method":3001,
  "dataType":"psn:sessionManager:sys:rps:members:deleted",
  "to":{
    "accountId":XXXXMASKINGTHISTOPROTECT MY DATA,
    "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
    "platform":[
      "REMOTE_PLAY"
    ],
    "isSystemOnly":false,
    "toSystem":false
  },
  "body":{
    "data":{
      "customProperties":{
        "from":{
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":XXXXMASKINGTHISTOPROTECT MY DATA",
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "platform":"REMOTE_PLAY"
        }
      },
      "members":[
        {
          "accountId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "deviceUniqueId":"XXXXMASKINGTHISTOPROTECT MY DATA",
          "onlineId":"XXXXMASKINGTHISTOPROTECT MY DATA,
          "platform":"REMOTE_PLAY"
        }
      ],
      "sessionId":"XXXXMASKINGTHISTOPROTECT MY DATA"
    }
  },
  "contexts":[
    {
      "contextId":"XXXXMASKINGTHISTOPROTECT MY DATA",
      "sequenceNumber":4
    }
  ]
}
[2024-05-13 11:58:57:112112] [V] Received notification of type 4
[2024-05-13 11:58:57:119119] [V] wait_for_notification: Found notification of type 4
[2024-05-13 11:58:57:119119] [V] Holepunch session state: 131195 = [ WS_OPEN DELETED CREATED CLIENT_JOINED DATA_SENT CONSOLE_JOINED CUSTOMDATA1_RECEIVED ]
[2024-05-13 11:58:57:119119] [I] chiaki_holepunch_session_fini: Holepunch session deleted.
[2024-05-13 11:58:57:119119] [E] websocket_thread_func: Select canceled.

**To Reproduce**
Basically what I wrote in the description

**Expected behavior**
It should be connecting since I followed the instructions

**Screenshots**
don't have screenshots

**Desktop (please complete the following information):**
 - OS: SteamOS 3.6 latest Beta Candidate/Preview build as of 13 May
 - Version SteamOS 3.6
 - Device Steam Deck Oled 1TB
 - Chiaki4deck Version 1.7.0

**Comment by L0WR1D3Rz** [comment]
Please Dev delete this issue

**Comment by L0WR1D3Rz** [comment]
Please delete
